torchsynth/module.py

```diff
@@ -586,6 +586,64 @@ def _forward(self) -> Signal:
         return ((1.0 + torch.cos(freqs)) / 2.0).as_subclass(Signal)
 
 
+class ModulationMixer(SynthModule):
```

Comment:
This is similar to the FaderBank mixer that I was working on in the FM branch, but modified to accept input signals and mix them together using the mix levels parameters.